i18n/en.json

```diff
@@ -2722,6 +2722,7 @@
   "post_message_view.edited": "(edited)",
   "post.ariaLabel.attachment": ", 1 attachment",
   "post.ariaLabel.attachmentMultiple": ", {attachmentCount} attachments",
+  "post.ariaLabel.emoji": "{name} emoji",
```

Comment:
not really sure if this is needed as the emoji names in itself are in english

Reply:
It's just so the label reads out as "smiley emoji" rather than just "smiley". I can change it to just the read the emoji's name instead if you'd like.

Reply:
@devinbinnie is not that.. is that there is no need to translate it, like "smiley emoticon" or who knows how "emoji"  is said in russian or whatever other language... if you know what I mean.

lets just add `{name } emoji` directly in the aria unless we make it so all the emoji names are translatable